src/app/utils/browsers.ts

```diff
@@ -32,4 +32,16 @@ export function isMac() {
   return mac;
 }
 
-export const isNativeDragAndDropSupported = () => 'draggable' in document.createElement('div');
+const android = navigator.platform.includes('Android');
+export function isAndroid() {
+  return android;
+}
```

Comment:
On my Android 11 with Firefox `navigator.platform` evaluates to `Linux aarch64` and with Chrome it evaluates to `Linux armv8l`, so this should probably test for `Linux` instead. That way drag and drop works on Chrome, but I can't get it to work on Firefox.

Reply:
Maybe I could test `userAgent` against `Android` then?

Reply:
That'd also work

Reply:
Actually, not sure if this is a common use case, but when people toggle the "Desktop site" setting in Firefox or Chrome on Android then the browsers spoof their User Agent to not include "Android", and in that case dnd won't work.

Reply:
That's a good point, but I'd rather be able to say "you shouldn't request the desktop site" than have things fail to work in Linux for some reason. Feels safe enough to mostly copy the logic at https://github.com/timruffles/mobile-drag-drop/blob/master/src/internal/feature-detection.ts